map-data/src/main/java/org/triplea/map/data/elements/PropertyList.java

```diff
@@ -0,0 +1,46 @@
+package org.triplea.map.data.elements;
+
+import java.util.List;
+import lombok.Getter;
+import org.triplea.generic.xml.reader.annotations.Attribute;
+import org.triplea.generic.xml.reader.annotations.BodyText;
+import org.triplea.generic.xml.reader.annotations.Tag;
+import org.triplea.generic.xml.reader.annotations.TagList;
+
+@Getter
+public class PropertyList {
+
+  @TagList private List<Property> properties;
+
+  @Getter
+  public static class Property {
+    @Attribute private java.lang.String name;
+
+    @Attribute(defaultValue = "false")
+    private java.lang.String editable;
+
+    @Attribute private java.lang.String player;
+
+    @Attribute private java.lang.String value;
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.String' due to existing implicit import 'java.lang.*'](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6131966)